CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ IMPROVEMENTS:
 
 BUG FIXES:
 * api: Fixed event stream connection initialization when there are no events to send [[GH-10637](https://github.com/hashicorp/nomad/issues/10637)]
-* cli: Fixed a bug where `quota status` and `namespace status` commands may panic if the CLI targets a pre-1.1.0 cluster
+* cli: Fixed a bug where `quota status` and `namespace status` commands may panic if the CLI targets a pre-1.1.0 cluster [GH-10620](https://github.com/hashicorp/nomad/pull/10620)
```

Comment:
should be in the form `[[]()]` otherwise changelogfmt gets sad

Reply:
D'oh